### PR TITLE
[Console] Migration plan - Part 2

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue
https://gravitee.atlassian.net/browse/APIM-312

## Description

- Various fix 
-  Implement secure step inputs with gv-sheam-form

## Additional context
image

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/apim-312-migrate-plan-p2/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-hnmicwyyyg.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts [hunk 1]
```diff
@@ -24,14 +57,81 @@ import { Subject } from 'rxjs';
...
+    });
+
+    this.secureForm
+      .get('securityTypes')
+      .valueChanges.pipe(
+        takeUntil(this.unsubscribe$),
+        tap(() => (this.securityConfigSchema = undefined)),
+        filter((securityType) => securityType !== PlanSecurityType.KEY_LESS),
+        map((securityType) => this.securityTypes.find((type) => type.id === securityType).policy),
+        switchMap((securityTypePolicy) => this.policyService.getSchema(securityTypePolicy)),
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts:100]: Do we need to add a catch error for this http request ?

File: gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts [hunk 2]
```diff
@@ -24,14 +57,81 @@ import { Subject } from 'rxjs';
...
+      )
+      .subscribe((schema) => {
+        this.securityConfigSchema = schema;
+      });
+
+    this.resourceService
+      .list({ expandSchema: false, expandIcon: true })
+      .pipe(takeUntil(this.unsubscribe$))
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts]: same here do we need a catch error here ?

**Review by Okhelifi — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts:100]: ✅

File: gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/portal/plans/plan/edit/2-secure-step/plan-edit-secure-step.component.ts]: ✅

**Review by phiz71 — APPROVED** [review]

**Pull request merged**